src/functionHandlers/notarisePdt/v2/handler.ts

```diff
@@ -64,6 +64,18 @@ export const notarisePdt = async (
   let signedEuHealthCerts: notarise.SignedEuHealthCert[] = [];
   if (config.isOfflineQrEnabled || whiteListNrics.includes(patientNricFin)) {
     try {
+      const testDataTypes = testData.map((test) => test.swabTypeCode);
+      if (
+        !testDataTypes.includes(config.swabTestTypes.ART) &&
+        !testDataTypes.includes(config.swabTestTypes.PCR)
+      ) {
+        throw new Error(
+          `signedEuHealthCerts not generated due to invalid swab Type Codes: ${JSON.stringify(
```

Comment:
Hmm here may not need the prefix since it's throwing an error (where it will be caught in line 95 right?)
```suggestion
          `Unsupported test type - ${JSON.stringify(
```

Reply:
Btw, instead of throwing error, should we just skip and do `traceWithRef()` instead?

Reply:
ok... i will change it.